src/Persistence.Tests/Yaml/DeserializerValidTests.cs

```diff
@@ -36,7 +50,8 @@ public class DeserializerValidTests : TestBase
         var deserializer = ServiceProvider.GetRequiredService<IYamlSerializationFactory>().CreateDeserializer(new() { IsTextFirst = true });
 
         var sut = deserializer.Deserialize<Control>(yaml);
-        sut.Should().NotBeNull().And.BeOfType<Screen>();
+        sut.ShouldNotBeNull();
```

Comment:
Is there a reason to split this into two lines? I think the test previously was doing exactly the same thing

Reply:
The 'ShouldNotBeNull' function is a custom extension that I created a bit ago that makes it possible to do null check validation in nullable contexts. Without this, the compiler complains about possible null dereference errors because the value returned from the previous line is nullable.

It's too bad that the FluentAssertions don't make it possible to tell the compiler about an assertion also ensuring nullable checks are done.